bot/exts/holidays/easter/easter_riddle.py

```diff
@@ -57,26 +60,33 @@ async def riddle(self, ctx: commands.Context) -> None:
 
         description = f"You have {TIMELIMIT} seconds before the first hint."
 
-        riddle_embed = discord.Embed(title=question, description=description, colour=Colours.pink)
-
-        await ctx.send(embed=riddle_embed)
-        await asyncio.sleep(TIMELIMIT)
-
-        hint_embed = discord.Embed(
-            title=f"Here's a hint: {hints[0]}!",
-            colour=Colours.pink
-        )
-
-        await ctx.send(embed=hint_embed)
-        await asyncio.sleep(TIMELIMIT)
-
-        hint_embed = discord.Embed(
-            title=f"Here's a hint: {hints[1]}!",
-            colour=Colours.pink
+        riddle_embed = discord.Embed(
+            title=question, description=description, colour=Colours.pink
         )
 
-        await ctx.send(embed=hint_embed)
-        await asyncio.sleep(TIMELIMIT)
+        await ctx.send(embed=riddle_embed)
+        hint_number = 0
+        while hint_number < 2:
+            try:
+                response = await self.bot.wait_for(
+                    "message",
+                    check=lambda m: m.channel == ctx.channel
+                    and m.author != self.bot.user,
+                    timeout=TIMELIMIT,
+                )
+                if response.content.lower() == self.correct.lower():
+                    self.winners.add(response.author.mention)
```

Comment:
Since there can only be one winner now, there's no need for it to be a list (and since it's only used in this function it doesn't need to be a class variable)